wordpress-mobile/WordPress-iOS
# Gutenberg private site image loader

This PR implements a React Native custom image loader to allow Gutenberg to load images from private sites.

To test:
 - Start a post on a private blog.
 - Add image blocks to the post using images from the media library.
 - Check if images are loaded.
 - Check the same in a regular public site.

Update release notes:
